src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -116,6 +117,35 @@ const SingleContentWrapper = styled.div`
   `}
 `;
 
+const OptionallyRenderedSkipWrapper = ({ skipLink, service, children }) =>
+  skipLink ? (
+    <SkipLinkWrapper service={service} {...skipLink}>
+      {children}
+    </SkipLinkWrapper>
+  ) : (
+    children
+  );
+
+const optionalSkipLinkProps = {
+  terms: shape({
+    '%title%': string,
+  }),
+  endTextVisuallyHidden: string,
+  endTextId: string,
+  text: string,
+};
+
+OptionallyRenderedSkipWrapper.propTypes = {
+  service: string,
```

Comment:
`service` is required. If you pass `skipLink`, but not `service`, this component will throw an error.